lib/logstash/inputs/kafka.rb

```diff
@@ -36,10 +36,21 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   # the same consumer group.
   config :group_id, :validate => :string, :default => 'logstash'
   # The topic to consume messages from
-  config :topic_id, :validate => :string, :required => true
+  config :topic_id, :validate => :string, :default => nil
+  # Whitelist of topics to include for consumption.
+  config :white_list, :validate => :string, :default => nil
+  # Blacklist of topics to exclude from consumption.
+  config :black_list, :validate => :string, :default => nil
+  # Indicate if `topic_id` is a blacklist to exclude from consumption.
+  config :blacklist_topics, :validate => :boolean, :default => false
```

Comment:
Now that we have explicit `white_list`/`black_list` params, can we remove this flag?